soul-common/src/main/java/org/dromara/soul/common/dto/convert/rule/RuleHandle.java

```diff
@@ -17,33 +17,25 @@
 
 package org.dromara.soul.common.dto.convert.rule;
 
-import lombok.Data;
-import org.dromara.soul.common.constant.Constants;
-
 import java.io.Serializable;
 
 /**
- * The type Divide rule handle.
+ * The RuleHandle interface.
  *
- * @author xiaoyu(Myth)
+ * @author yiwenlong (wlong.yi@gmail.com)
  */
-@Data
-public class DivideRuleHandle implements Serializable {
+public interface RuleHandle extends Serializable {
 
     /**
-     * loadBalance.
-     * {@linkplain org.dromara.soul.common.enums.LoadBalanceEnum}
+     * It will be call when RuleHandleFactory to generate a RuleHandle object.
+     * @param path  path.
+     * @return      it's self.
      */
-    private String loadBalance;
+    RuleHandle init(String path);
```

Comment:
the method is createDefualt(String path)

Reply:
Do you mean it is more appropriate to use `createDefualt` instead of `init` as the method name?